src/js/components/Field/Field.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { getClassName } from '../../utils/helpers'
+
+class Field extends PureComponent {
+  renderSuccess = () => (
+    <span className='FormGroup-validation'>
+      <Icon name='check' />
+    </span>
+  )
+
+  render () {
+    const { children, status, className, mods, style } = this.props
+
+    const fieldClasses = getClassName(
+      className,
+      status === 'error' && 'is-notValid',
+      status === 'success' && 'is-valid',
+      mods
+    )
+
+    return (
+      <div className={ fieldClasses } style={ style }>
+        { status === 'success' && this.renderSuccess() }
+        { children }
+      </div>
+    )
+  }
+}
+
+Field.propTypes = {
+  children: PropTypes.node.isRequired,
```

Comment:
Would this support multiple children? For example in the case of multiple checkboxes or radios that have validation that applies to them as a group (ie they are required). Example that shows the `FormGroup` pattern with two nested radio buttons that are required:

![form-group-multiple-children](https://user-images.githubusercontent.com/9888467/37541923-4f89fe4a-292a-11e8-8e0f-e05810d7782d.gif)

Reply:
This should work like you said, you could pass 2 checkboxes say inline that would render right next to each other under a single label and with a single set of errors.